server/api/utils/pagination.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import typing
+
+import sqlalchemy.orm
+
+import mlrun.common.schemas
+import mlrun.errors
+import mlrun.utils.singleton
+import server.api.crud
+from mlrun.utils import logger
+
+
+class PaginatedMethods:
+    _methods: list[typing.Callable] = [
+        # TODO: add methods when they implement pagination
+        # server.api.crud.Runs().list_runs,
+    ]
+    _method_map = {method.__name__: method for method in _methods}
```

Comment:
If you implemented `get_name`, why not use it here?

Reply:
Sure 👍

Reply:
Spoke offline removed get_name